workbench-core/datasets/src/dataSetService.test.ts

```diff
@@ -70,6 +75,8 @@ describe('DataSetService', () => {
     id: mockUserId,
     roles: []
   };
+  const mockCreatedAt = new Date().toISOString();
+  const mockAccessLevel: DataSetsAccessLevel = 'read-only';
```

Comment:
NOT BLOCKING: you could call this `mockReadOnlyAccessLevel` and it would never be confused when a `read-write` is added.

Reply:
I'll make this change with my listDatasets PR

Reply:
need to fix merge conflicts, so I'll change now